homework/content.js

```diff
@@ -1,37 +1,68 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+/**
+ * @typedef {Object} Note
+ * @property {String} title
+ * @property {String} passages
+ */
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
+(function(exports) {
+
+  /**
+   * Create a instance of ContentManager
+   *
+   * @constructor
+   * @this {ContentManager}
+   */
+  var ContentManager = function () {
+    this._wrapper = document.querySelector('#note-content-wrapper');
+  }
+
+  /**
+   * Add an event handler for note-open event which draw correct note contents
+   *
+   * @this {ContentManager}
+   */
+  ContentManager.prototype.start = function () {
+    window.addEventListener('note-open', (function(event) {
```

Comment:
See the second code example in:

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#The_value_of_this_within_the_handler

That's what I asked you to bind `this`: via call `addEventListener` with an object instance, its `handleEvent` will be automatically bound with the object you wish as its `this`.